scipy/cluster/tests/test_vq.py

```diff
@@ -385,20 +352,16 @@ def test_kmeans2_kpp_high_dim(self, xp):
         ])
 
         data = xp.asarray(data)
-        res, _ = kmeans2(data, xp.asarray(2), minit='++')
+        res, _ = kmeans2(data, 2, minit='++')
```

Comment:
to clarify, these are simplifications and the test would pass either way?

Reply:
Yes. I suppose we could test both integer and array input.